--- Coin-3.1.3/src/profiler/SoProfiler.cpp.orig	2010-03-02 14:20:09.000000000 +0100
+++ Coin-3.1.3/src/profiler/SoProfiler.cpp	2014-11-04 20:53:37.368850092 +0100
@@ -552,9 +552,9 @@
     // send ansi-console clear screen code
     static const char * CLEAR_SEQUENCE = "\033c";
     if (profiler::console::onstdout) {
-      fprintf(coin_get_stdout(), CLEAR_SEQUENCE);
+      fputs(CLEAR_SEQUENCE, coin_get_stdout());
     } else if (profiler::console::onstderr) {
-      fprintf(coin_get_stderr(), CLEAR_SEQUENCE);
+      fputs(CLEAR_SEQUENCE,coin_get_stderr());
     }
   }
 
